delegation-verify-coordinator/templates/deployment.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "delegation-verify-coordinator.fullname" . }}
+  labels:
+    {{- include "delegation-verify-coordinator.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "delegation-verify-coordinator.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "delegation-verify-coordinator.labels" . | nindent 8 }}
+	      {{- with .Values.podLabels }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      serviceAccountName: {{ include "delegation-verify-coordinator.serviceAccountName" . }}
+      containers:
+        - name: {{ .Chart.Name }}
```

Comment:
I would not use a release name as a container name. This only appears in a context of a pod so it does not need to be that much unique. You only want this container to be distinguishable from init containers or any sidecars if there are any. Instead I would advice to have a one word name. Makes it much nicer to work with with kubectl or grafana.

Reply:
+1